setup.py install ignores previous built_ext -I arguments

Had to compile current proj myself, now installing cartopy fails with setup.py install not finding proj's header files.
The initial setup.py build_ext --inplace -I$HOME/local/include -L$HOME/local/lib works, but the -I flag cannot be passed to setup.py install in any way I have found. Now install tries to run this:
    lib/cartopy/trace.cpp -o build/temp.freebsd-10.3-RELEASE-p4-amd64-3.5/lib/cartopy/trace.o
which of course errors out:
    lib/cartopy/trace.cpp:282:10: fatal error: 'proj_api.h' file not found
Manually running this with a proper -I added works, but setup.py install nevertheless tries to compile it again without -I

--- a/INSTALL
+++ b/INSTALL
@@ -29,7 +29,7 @@ Once you have satisfied the requirements detailed below, simply run::
 For non-standard locations, additional build lib & include paths
 can be provided as per-usual at build_ext phase::
 
-    python setup.py build_ext --inplace -I/path/to/include -L/path/to/lib
+    python setup.py build_ext -I/path/to/include -L/path/to/lib
     python setup.py install
 
 

